index.js

```diff
@@ -57,20 +57,31 @@ const toUrlFormat = (item) => {
 
 const exec = (cmd, args = []) =>
   new Promise((resolve, reject) => {
-    const app = spawn(cmd, args, { stdio: "pipe" });
+    const app = spawn(cmd, args);
```

Comment:
`options.stdio` defaults to `pipe`, [Ref](https://nodejs.org/api/child_process.html#optionsstdio)